CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## TS CLI&SDK [2.1.0](https://github.com/marinade-finance/validator-bonds/compare/v2.0.6...v2.1.0) (2025-01-31)
+## TS CLI&SDK [2.1.2](https://github.com/marinade-finance/validator-bonds/compare/v2.1.0...v2.1.2) (2025-02-04)
+
+### Fixes
+
+Fix of wrong npm registry deployment
+
+## TS CLI&SDK [2.1.0](https://github.com/marinade-finance/validator-bonds/compare/v2.0.6...v2.1.0) (2025-02-04)
```

Comment:
Is the date changed from `2025-01-31` to `2025-02-04` for this older log on purpose?

Reply:
Yes, that was an errror of the changelog data. The deployment was considered to happen but as the PR was not reviewed last week I did not publish it. So it is just aligning the doc with reality.